### PR TITLE
Feature:  Home Page

### PR DESCRIPTION
**Pull request by ssimmons05**
# [Create Home Page](https://trello.com/c/FN0n6LU6)

## A short description of what this PR does.

As a user I'd like to see a countdown to the Hacktoberfest SnapIT event.

## How was it tested?
This was linted with Yarn and tested with Jest.
home_page-jest_test

### Yarn Lint
home_page-lint_test
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

**Review by aaronshivers — CHANGES_REQUESTED** [review]
🔧 It looks like you created this branch from your intl branch. Can you rebase with the upstream and remove all changes not related to adding the home page?

**Comment by ssimmons05** [comment]
> 🔧 It looks like you created this branch from your intl branch. Can you rebase with the upstream and remove all changes not related to adding the home page?

Addressed: https://github.com/aaron-org/snapit-hacktoberfest/pull/23/commits/b382a940488bbeed643fc6ad79ed21ab864953f8

File: src/App.css [hunk 1]
```diff
@@ -16,7 +16,8 @@ h1 {
 }
 
 ul {
-  margin-top: 40px;
+  margin-top: 0;
+  padding: 0;
```
Comment (aaronshivers) [inline src/App.css:20]: :wrench: Let's use margin and padding from material instead.

File: src/App.test.jsx [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import React from "react";
 import { shallow } from "enzyme";
 import App from "./App";
+import FAQ from "./pages/FAQ";
+import Information from "./pages/Information";
+import Profile from "./pages/Profile";
+import Registration from "./pages/Registration";
+import AccordionList from "./components/AccordionList";
```
Comment (aaronshivers) [inline src/App.test.jsx:8]: :wrench: Rebase this branch and remove these lines.

File: src/App.test.jsx [hunk 3]
```diff
@@ -10,6 +15,39 @@ test("renders without error", () => {
...
+
+test("renders Accordion List", () => {
+  const wrapper = shallow(<AccordionList />);
+  const accordionListComponent = findByTestAttr(
+    wrapper,
+    "component-accordionList"
+  );
+  expect(accordionListComponent.length).toBe(1);
+});
+
```
Comment (aaronshivers) [inline src/App.test.jsx:50]: :wrench: Rebase this branch and remove these lines.

File: src/components/CountdownTimer.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
```
Comment (aaronshivers) [inline src/components/CountdownTimer.jsx:1]: :wrench: This needs to be in its own directory.

File: src/components/CountdownTimer.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+const CountdownTimer = () => {
+  const [, setUpdate] = useState(0);
```
Comment (aaronshivers) [inline src/components/CountdownTimer.jsx:4]: :question: Why are we setting this variable if we aren't using it?

File: src/pages/Home.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+import CountdownTimer from "../components/CountdownTimer";
+
+const Home = () => (
+  <div className="Home" data-test="Home-page">
+    <h1>SnapIT Hacktoberfest Starts In:</h1>
+    <CountdownTimer data-test="countdown-display" />
+  </div>
+);
+
+export default Home;
```
Comment (aaronshivers) [inline src/pages/Home.jsx:11]: :wrench: This file should be removed.

File: src/pages/Home.test.jsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  const wrapper = shallow(<Home />);
+  const homeComponent = findByTestAttr(wrapper, "Home-page");
+  expect(homeComponent.length).toBe(1);
+});
+
+test("renders CountdownTimer", () => {
+  const wrapper = shallow(<Home />);
+  const countdownDisplay = findByTestAttr(wrapper, "countdown-display");
+  expect(countdownDisplay.length).toBe(1);
+});
```
Comment (aaronshivers) [inline src/pages/Home.test.jsx:17]: :wrench: remove this file

File: src/pages/Home/Home.jsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import CountdownTimer from "../../components/CountdownTimer";
+
+const Home = () => (
+  <div className="Home" data-test="Home-page">
+    <h1>SnapIT Hacktoberfest Starts In:</h1>
```
Comment (aaronshivers) [inline src/pages/Home/Home.jsx:6]: :wrench: add a test for this heading

File: src/pages/Information/Information.jsx [hunk 9]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">What is Hacktoberfest</h2>
```
Comment (aaronshivers) [inline src/pages/Information/Information.jsx:6]: :question: Did you mean to remove the question mark?

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 10]
```diff
@@ -30,3 +31,38 @@ li {
...
+  padding-top: 20px;
+}
+
+#labels li:first-child {
+  border-top-left-radius: 7px;
+}
+
+#labels li:last-child {
+  border-top-right-radius: 7px;
+}
```
[the remark this answers is not in the document]
Comment (ssimmons05) [reply src/App.css]: Addressed:  f639309aca79206b6c8cc424c70be42e6adfb37b

File: src/components/CountdownTimer.jsx (hunk 4 above)
Comment (ssimmons05) [reply src/components/CountdownTimer.jsx]: Addressed:  0141e30395a7c95559f408514669b8dae7d1439f

File: src/App.test.jsx (hunk 2 above)
Comment (ssimmons05) [reply src/App.test.jsx]: Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b

File: src/App.test.jsx (hunk 3 above)
Comment (ssimmons05) [reply src/App.test.jsx]: Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b

File: src/pages/Information/Information.jsx (hunk 9 above)
Comment (ssimmons05) [reply src/pages/Information/Information.jsx]: Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b

File: src/App.css (hunk 1 above)
Comment (ssimmons05) [reply src/App.css]: Addressed:  2c594aaa7d40e5110355ebdfb9a319ce72161bd8

File: src/pages/Home.jsx (hunk 6 above)
Comment (ssimmons05) [reply src/pages/Home.jsx]: Addressed:  f639309aca79206b6c8cc424c70be42e6adfb37b

File: src/pages/Home.test.jsx (hunk 7 above)
Comment (ssimmons05) [reply src/pages/Home.test.jsx]: Addressed:  f639309aca79206b6c8cc424c70be42e6adfb37b

File: src/components/CountdownTimer.jsx (hunk 5 above)
Comment (ssimmons05) [reply src/components/CountdownTimer.jsx]: Discussed on Friday.

File: src/pages/Home/Home.jsx (hunk 8 above)
Comment (ssimmons05) [reply src/pages/Home/Home.jsx]: Addressed:  52d3b149150f2810f78e3a2c2c6780dd74185b0c

File: src/App.css [hunk 11]
```diff
@@ -1,32 +1,7 @@
...
-  color: orange;
-  display: inline-flex;
-  list-style: none;
-  padding: 0 30px;
-  width: 120px;
-}
-
 a:link {
   text-decoration: none;
 }
```
Comment (aaronshivers) [inline src/App.css:7]: ❓ Are we still using this file for anything? If not, can you remove it?

File: src/components/Navigation/Navigation.test.jsx [hunk 12]
```diff
@@ -1,32 +0,0 @@
...
-  });
-
-  test("clicking button displays navigation menu", () => {
-    const wrapper = setup();
-    const button = findByTestAttribute(wrapper, "nav-button");
-    button.simulate("click");
-    const listDisplay = findByTestAttribute(wrapper, "list-display");
-    expect(listDisplay.length).toBe(1);
-  });
-});
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.test.jsx:32]: ❓ I just wanted to make sure this was deleted on purpose.

File: src/pages/FAQ.jsx [hunk 13]
```diff
@@ -1,11 +1,14 @@
...
 
 const FAQ = () => (
-  <div className="FAQ" data-test="FAQ-page">
-    <h1>FAQ</h1>
-    <AccordionList />
-  </div>
+  <Box mt="40" mx="auto ">
+    <div className="FAQ" data-test="FAQ-page">
+      <h1>FAQ</h1>
+      <AccordionList />
```
Comment (aaronshivers) [inline src/pages/FAQ.jsx:9]: 🔧 add tests for these items

File: src/pages/Information/Information.jsx [hunk 14]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">What is Hacktoberfest?</h2>
```
Comment (aaronshivers) [inline src/pages/Information/Information.jsx:6]: ⛏️ Maybe go with the old version here.

File: src/pages/Profile.jsx [hunk 15]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
+import Box from "@material-ui/core/Box";
 
 const Profile = () => (
-  <div className="Profile" data-test="Profile-page">
-    <h1>Profile</h1>
-  </div>
+  <Box mt="40" mx="auto ">
+    <div className="Profile" data-test="Profile-page">
+      <h1>Profile</h1>
```
Comment (aaronshivers) [inline src/pages/Profile.jsx:7]: 🔧 add test for the heading

File: src/pages/Registration.jsx [hunk 16]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
+import Box from "@material-ui/core/Box";
 
 const Registration = () => (
-  <div className="Registration" data-test="Registration-page">
-    <h1>Registration</h1>
-  </div>
+  <Box mt="40" mx="auto ">
+    <div className="Registration" data-test="Registration-page">
+      <h1>Registration</h1>
```
Comment (aaronshivers) [inline src/pages/Registration.jsx:7]: 🔧 test

**Review by aaronshivers — CHANGES_REQUESTED** [review]
🐸 Great job so far!

File: src/pages/Profile.jsx (hunk 15 above)
Comment (ssimmons05) [reply src/pages/Profile.jsx:7]: This component no longer contains this element per latest commit:  c0dd94230ccd7f24127e61d1966455951fa0dd42

File: src/pages/Registration.jsx (hunk 16 above)
Comment (ssimmons05) [reply src/pages/Registration.jsx:7]: Addressed:  206bddd888d20d4c1fa834eea21e5795d847cdf2

File: src/pages/FAQ.jsx (hunk 13 above)
Comment (ssimmons05) [reply src/pages/FAQ.jsx]: Addressed: e67701b768282739f6b632184983497816fa6ab0

File: src/components/Navigation/Navigation.test.jsx (hunk 12 above)
Comment (ssimmons05) [reply src/components/Navigation/Navigation.test.jsx]: Addressed:  e67701b768282739f6b632184983497816fa6ab0

File: src/pages/Information/Information.jsx (hunk 14 above)
Comment (ssimmons05) [reply src/pages/Information/Information.jsx]: I removed the question mark, to match previous version.  Let me know if I misunderstood.  
Commit:  4f5d0c2f70b91c4f8f4190a7289064862eac6bfc

File: src/App.css (hunk 11 above)
Comment (ssimmons05) [reply src/App.css:7]: I removed the file but haven't found a way to style this with native material ui props yet.  Creating an issue for this to come back to.

**Review by aaronshivers — CHANGES_REQUESTED** [review]
🐷 🐴 🐘

File: src/pages/FAQ/FAQ.test.jsx [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+    const countdownDisplay = findByTestAttr(wrapper, "FAQ-list");
+    expect(countdownDisplay.length).toBe(1);
+  });
+
+  test("renders FAQ heading", () => {
+    const wrapper = setup();
+    const countdownDisplay = findByTestAttr(wrapper, "heading-display");
+    expect(countdownDisplay.length).toBe(1);
+  });
+});
```
[the remark this answers is not in the document]
Comment (ssimmons05) [reply src/pages/FAQ/FAQ.test.jsx]: Addressed:  bec8571305a0d4b64ca051dac06ad628bb7f4af4

**Review by aaronshivers — APPROVED** [review]

**Pull request merged**